packages/app/lib/main.dart

```diff
@@ -11,6 +14,16 @@ import 'src/app/bootstrap.dart';
 ///
 /// This uses [Bootstrap] to launch [App].
 Future<void> main() async {
+  //Set minimum window size for desktop
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
This is already called in `App#bootstrap()`, could you move the rest of the code over there?